README.rst

```diff
@@ -98,13 +98,21 @@ Quick Example
 
     bot = commands.Bot(command_prefix='$')
 
+
     @bot.command()
     async def ping(ctx):
         await ctx.reply('Pong!')
 
+    @bot.command()
+    async def say(ctx, *, arg):
+         await ctx.message.delete()
+         await ctx.send(f"{ctx.author.mention}'s message: {arg}")
```

Comment:
either edit the quotes to be single here or double in the rest (latter preferred)